tests/Areas/AzureTerraformBestPractices/UnitTests/AzureTerraformBestPracticesGetCommandTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.CommandLine.Parsing;
+using System.Text.Json;
+using AzureMcp.Areas.AzureTerraformBestPractices.Commands;
+using AzureMcp.Models.Command;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NSubstitute;
+using Xunit;
+
+namespace AzureMcp.Tests.Areas.AzureTerraformBestPractices.UnitTests;
+
+[Trait("Area", "AzureTerraformBestPractices")]
+public class AzureTerraformBestPracticesGetCommandTests
+{
+    private readonly IServiceProvider _serviceProvider;
+    private readonly ILogger<AzureTerraformBestPracticesGetCommand> _logger;
+    private readonly CommandContext _context;
+    private readonly AzureTerraformBestPracticesGetCommand _command;
+    private readonly Parser _parser;
+
+    public AzureTerraformBestPracticesGetCommandTests()
+    {
+        var collection = new ServiceCollection();
+        _serviceProvider = collection.BuildServiceProvider();
+
+        _context = new(_serviceProvider);
+        _logger = Substitute.For<ILogger<AzureTerraformBestPracticesGetCommand>>();
+        _command = new(_logger);
+        _parser = new(_command.GetCommand());
+    }
+
+    [Fact]
+    public async Task ExecuteAsync_ReturnsAzureTerraformBestPractices()
+    {
+        string expectedAzureTerraformBestPractices = """
+- Ensure the request is for Azure resources. If not, do not call this tool.
+- Check for Terraform installation on the first Terraform-related request in a session
+- Provide the winget installation command if needed: winget install Hashicorp.Terraform
+- Follow the best practices style guide in the link https://developer.hashicorp.com/terraform/language/style before generating the Terraform code.
```

Comment:
Does providing a link actually work? I'm unsure if the agent will be capable to load the website and read its contents.